db/scripts/01_cassandra_queue.cql

```diff
@@ -1,25 +1,33 @@
 CREATE TABLE queue (
-  queuename text PRIMARY KEY,
+  queueid text PRIMARY KEY,
+  queuename text,
   bucket_size int,
-  max_dequeue_count int,
-  status text
+  max_delivery_count int,
+  status int,
```

Comment:
i'm not sure why both queue and queue name manager need a status and version. we could probably even remove queue name fro this table.